com.unity.netcode.gameobjects/Runtime/Serialization/ISerializeByMemcpy.cs

```diff
@@ -0,0 +1,7 @@
+
+namespace Unity.Netcode
+{
+    public interface ISerializeByMemcpy
```

Comment:
```suggestion
    public interface INetworkSerializeByMemcpy
```

Reply:
Are you at all worried this will make it easier to confuse this with `INetworkSerializable`?

Reply:
I see more explicit intention by the name & consistency in `INetworkSerializeByMemcpy`

Reply:
On one hand, it does live here: `Unity.netcode.ISerializeByMemcpy`
Which provides an understood intention.
On the other hand, `INetworkSerializeByMemcpy` does help further define that it is network serialization specific..
¯\_ (ツ)_/¯

Reply:
Where it lives doesn't really matter to me. At a glance, if I see `INetworkSerializeByMemcpy`, I know automatically it's for Networking. If I see `ISerializeByMemcpy` I might assume it's a generic thing for any and all scenario. And since we already have a `INetworkSerializable` for consistency and to fit, I would name it `INetworkSerializeByMemcpy`. I thing both are distinct enough to not be confused but understanding they are under the same umbrella.